sdk/core/core-amqp/src/cbs.ts

```diff
@@ -129,15 +129,15 @@ export class CbsClient {
         );
         log.cbs(
           "[%s] Successfully created the cbs sender '%s' and receiver '%s' " +
-            "links over cbs session.",
```

Comment:
Looks like Emacs decided to eat some indentation, I'll revert the whitespace changes before commit